### ISSUE TITLE
Broken with Security Update 2018-002 High Sierra

### ISSUE THREAD

**Issue by dactylo** [JK3Y/asepsis#9]
Seems this doesn't work with Security Update 2018-002 High Sierra (released Oct 30).

**rake build** supposedly succeeds, though there are messages from cp about missing AsepsisUpdater.app etc.

**rake install** ends with the following:

```
> "/Library/Application Support/Asepsis/ctl/asepsisctl" install_wrapper
the original library is expected to be a fat binary. (/System/Library/PrivateFrameworks/DesktopServicesPriv.framework/Versions/A/DesktopServicesPriv should contain both slices x86_64 and i386)
> "/Library/Application Support/Asepsis/ctl/asepsisctl" install_updater
> sudo cp "/Library/Application Support/Asepsis/com.binaryage.asepsis.updater.plist" "/Library/LaunchAgents/com.binaryage.asepsis.updater.plist"
Asepsis installation encountered some failures, please inspect the command output.
```

**asepsisctl diagnose** says the following:

```
DesktopServicesPriv (/System/Library/PrivateFrameworks/DesktopServicesPriv.framework/Versions/A/DesktopServicesPriv) is not properly installed.
  => Have you installed a system update recently? It might revert it back to the original version.
```

**Comment by xiaozhuai** [comment]
It is because of that you install a system update recently. Then you should reinstall asepsis.
To do this
```bash
asepsisctl uninstall_wrapper
asepsisctl install # or rake install for your case
asepsisctl diagnose
```
then find .DS_Store and delete it 
```bash
sudo find / -name .DS_Store -delete
```
Finally reboot your mac.
Hope that helps : )

**Comment by dactylo** [comment]
Thanks for try to help. There were 2 issues:

1. That error was because **is_library_fat_with_both_slices()** was failing, because I had a different "file" command in my path, with a different output. I hardcoded /usr/bin/file.
2. Then **codesign** was failing with "code object is not signed at all". I added --deep and that seemed to fix the issue, though I'm not sure what other ramifications that may have.
